api/v1/trigger_types.go

```diff
@@ -121,6 +127,20 @@ type KubernetesEventTemplateRegexp struct {
 	Source corev1.EventSource `json:"source,omitempty"`
 }
 
+// CronTemplate specifies the template to create a diagnosis periodically at fixed times.
+type CronTemplate struct {
+	// Schedule is the schedule in cron format.
+	// See https://en.wikipedia.org/wiki/Cron for more details.
+	Schedule string `json:"schedule"`
+}
+
+// TriggerStatus defines the observed state of Trigger.
+type TriggerStatus struct {
+	// LastScheduleTime is the last time the cron was successfully scheduled.
+	// +optional
+	LastScheduleTime *metav1.Time `json:"lastScheduleTime,omitempty"`
```

Comment:
I think `LastScheduleTime` can be very useful in Cron task, every trigger template should update this parameter, if a Trigger sets Prometheus Alert and Cron both, and Prometheus Alert is triggered within a minute, then Cron will not trigger at this point in time.

Reply:
It should be a convention that the `LastScheduleTime` of a diagnosis is maintained by a trigger. However, Allowing various triggers to update the status of a diagnosis would break the independency of various triggers. This dependency might introduces complexity and surprise in intermediating triggers.